src/App.scss

```diff
@@ -0,0 +1,111 @@
+.Landing {
+  display: flex;
+  flex: 1;
+  flex-wrap: wrap;
+  height: 100vh;
+}
+
+.Login {
+  align-items: center;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  font-family: 'Open Sans', sans-serif;
```

Comment:
is this the default font family? perhaps it's worth putting it into a constant like `$default-font`. Also you might want to do `body { font-family: $default-font } ` so you don't have to set it for each container.

Reply:
Good idea! 👍